sdk/src/main/java/com/hedera/hashgraph/sdk/ManagedNode.java

```diff
@@ -5,10 +5,12 @@
 
 import java.util.concurrent.ExecutorService;
 import java.util.concurrent.TimeUnit;
+import javax.annotation.Nullable;
 
 abstract class ManagedNode {
     String address;
-    ManagedChannel channel;
+    @Nullable
```

Comment:
Personal preference, but can you add a new line before and after the field with an annotation; I think that's more readable that way
```java
String address;

@Nullable
ManagedChannel channel = null;

final ExecutorService executor;
```